test/common/testlib.py

```diff
@@ -1409,25 +1409,45 @@ def jsquote(str):
     return json.dumps(str)
 
 
+def set_obj_attribute(testEntity, attribute, value, raise_text="", base_class=unittest.TestCase):
```

Comment:
Do you mind to rename `raise_text` to `decorator_name` to clarify what it does?

Reply:
this is probably not relevant after rewrote to the more generic structure.